advice/steve-m-advice.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="../css/steve-m-advice.css">
+        <!-- replace Skillcrush with your name here -->
+        <title>Steve Manning's Advice For Newbies</title>
+    </head>
+
+    <!-- replace skillcrush with your name here -->
+    <body class="skillcrush-advice">
+        <header>
+            <a href="../index.html">Back to Main Page</a>
+            <h1>Hi my name is Steve Manning and I have some advice for you!</h1>
+        </header>
+
+        <section>
+            <div class="advice">
+                <h4>Confusion is your friend!</h4>
+                <p>When you start programming your first web app you are doing something that you have never done before. It's a new challenge, a new opportunity, and in the case of software development, something so specific that it's probably never been done before.</p>
+
+                <p>Confusion, therefore, is a sign that you are heading in the right direction. It's a signal that you have engaged in the problem solving pursuit that is math and technology and that it's time for you to dig deeper and keep going because clarity awaits.</p>
+
+                <h4>Suspended disbelief plays an important role.</h4>
+                <p>When you are learning a new technology, whether you are learning to write your first programming language, or just the latest framework, you go through this uncomfortable phase where you know that the code you are looking at is doing this THING but you don't understand HOW exactly it's happening.</p>
+
+                <p>This is a normal part of the process of learning a new technology, and in fact, it's a critical part of the process.</p>
+
+                <p>You see when you encounter a new programming language part of your learning process includes learning to train your eye. The way to train your eye is to read a good deal of code even when you don't totally understand what it's doing.</p>
+            </div>
+        </section>
+
+        <footer>
+            <!-- replace Skillcrush with your name here -->
+            <p>© Copyright Steve Manning 2018.</p>
+        </footer>
+    </body>
+</html>
```

Comment:
please use newlines at end of files

Reply:
newlines? I'm not absolutely sure I understand what you mean by that, can you please elaborate more on exactly what that means?

Reply:
would you please read this question on  [stack overflow](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
I've read and re-read it a few times and I'm still confused on what you're telling me to do. Is it adding a <br> line-break at the end of my footer paragraph?